### DISCUSSION TITLE
gray-scale for website

### DISCUSSION THREAD

**Discussion by AveyBD** [plateaukao/einkbro#293] [Ideas]
Hello Eink Bro developers and community,

I hope this message finds you well. I'd like to propose a feature that I believe would greatly enhance the usability of Eink Bro - the introduction of a Grayscale Mode.

## The Importance of Grayscale Mode
Eink Bro is already an incredible tool for enhancing the web browsing experience on E Ink displays. However, there's one area where it could become even better - readability.

## Inspire and Motivate
I believe that by incorporating a Grayscale Mode into Eink Bro, we would not only improve the software but also promote healthier and more accessible web browsing. This feature aligns perfectly with the core values of E Ink technology - reduced eye strain, energy efficiency, and readability.

Let's work together to make Eink Bro an even more outstanding tool for users of E Ink devices. Your contribution to this project has the potential to positively impact the lives of countless individuals who rely on these displays for their daily tasks.

I'm excited about the possibilities this feature can bring to Eink Bro, and I'm looking forward to collaborating with the talented Eink Bro community to make it happen.

Thank you for considering this feature request!

Best regards,
Debashish D. Dev

**Comment by plateaukao** [comment]
Hi, @AveyBD 
I supported this feature with another app that works for the whole device. You can check it out here: https://github.com/plateaukao/quickrotate/releases/tag/v2.2.0
